app/main.py

```diff
@@ -2,0 +1,17 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
```

Comment:
The condition correctly checks for the command format. Ensure that the error handling or logging is appropriate if the command is invalid.